api/envoy/api/v2/rds.proto

```diff
@@ -53,7 +61,10 @@ message RouteConfiguration {
   string name = 1;
 
   // An array of virtual hosts that make up the route table.
-  repeated route.VirtualHost virtual_hosts = 2 [(gogoproto.nullable) = false];
+  repeated route.VirtualHost virtual_hosts = 2; [(gogoproto.nullable) = false];
```

Comment:
I think this should be nullable. If VHDS is enabled, it makes sense for virtual host updates to come over VHDS while route configs (minus virtual hosts) come across RDS.

Reply:
Should drop the semicoolon after virtual_hosts =2

Reply:
Agree, removed that.